pkg/templates/charts/toggle/grc/templates/grc-policy-addon-clusterrole.yaml

```diff
@@ -68,7 +66,6 @@ rules:
   - cert-policy-controller
   - config-policy-controller
   - governance-policy-framework
-  - iam-policy-controller
```

Comment:
Should this particular deletion RBAC be left in the addon-controller permissions for cleanup?

Reply:
@dhaiducek I don't it's necessary since the MCH operator will delete the ClusterManagementAddOn which deletes the ManagedClusterAddOn objects which deletes the ManifestWork (because of owner references).

The addon controller will not be aware that iam-policy-controller ever existed in 2.11.

Reply:
Addon-contorller clean iam? or mce does?

Reply:
Multiclusterhub operator cleans up the CMA as you have and Kubernetes should handle the rest.

Reply:
https://kubernetes.io/docs/concepts/architecture/garbage-collection/#orphaned-dependents